cobbler/tftpgen.py

```diff
@@ -1063,9 +1063,16 @@ def generate_script(self, what, objname, script_name):
         else:
             blended['img_path'] = os.path.join("/images", distro.name)
 
-        template = os.path.normpath(os.path.join("/var/lib/cobbler/autoinstall_scripts", script_name))
```

Comment:
When `generate_scripts` was introduced (I believe here: https://github.com/cobbler/cobbler/commit/955568d075abea57ec9d3cff01eb80a6eb37482f#diff-7ae3873d5c766467d4634ede08a8e6d6R1037) the path was `/var/lib/cobbler/scripts` and it actually existed. The new path `/var/lib/cobbler/autoinstall_scripts` does not exist.
Is this intended?

Reply:
In fact there still are another references to /var/lib/cobbler/autoinstall_scripts
MANIFEST.in:recursive-include autoinstall_scripts *
setup.py:            ("%s/scripts" % libpath, glob("autoinstall_scripts/*")),
Without checking the whole git history.., it looks as if a setup.py install should create the directory or the *.spec file and autoinstall_scripts (former kickstart_* functionality naming) has been split out from other scripts?
Not sure, but if, then the other autoinstall_scripts occurences should be taken care for too, to aovid other leftovers.

BTW: What is your problem/error?
Maybe simply creating the directory solves things. Then the correct fix might be to create the dir in the .spec file?